english/lunge-for-the-lighter/staircase/staircase.md

```diff
@@ -0,0 +1 @@
+You run up the staircase hoping to find some solace and a chance to catch your breath. You reach the top and to your relief find your feet land on a reassuringly solid parquet floor. You pause to pick clumps of marshmallow from the tread in your boots and contemplate your new situation. You find yourself looking around and realise that you're now standing in the living room of a penthouse apartment. On the wall, hangs a picture of a man with a mullet sporting a pair of aviators, and next to it hangs a calendar reading "31st December 1985". Ahead of you there is a corridor lined with a series of white marble sculptures. To your right you notice a fluttering curtain and see that the roof terrace door is slightly ajar. You pick up a Mars Bar laying that's on the coffee table, peel open the wrapper and decisively bite the end of the bar, covering for your indecision over which direction to explore.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.